tests/fake_lab/__init__.py

```diff
@@ -24,7 +33,7 @@
 add_standalone_sample_position(
     SamplePosition(
         "furnace_temp",
-        number=4,
+        number=64,
```

Comment:
could you please explain me what this number represents and add a comment?

Reply:
is like 64 max positions? and why it was just 4 before?

Reply:
This represents the number of sample positions in this spot. We increase the number because now we use more complicated experiments in the unittest.